debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4

```diff
@@ -672,10 +672,9 @@ dropIndex
       uid ON tableName
       (
         ALGORITHM '='? algType=(DEFAULT | INPLACE | COPY)
-      )?
```

Comment:
Isn't it a bit too lenient now? IOW, the one before looked right (in contrast to the other usage above), it allowed ALGORITHM and LOCK, no?

Reply:
But it prescribed order which is no strict